packages/cli/lib/services/parser.service.ts

```diff
@@ -7,14 +7,31 @@ import parser from '@babel/parser';
 import { SyncConfigType } from '@nangohq/shared';
 
 class ParserService {
-    public callsAreUsedCorrectly(filePath: string, type = SyncConfigType.SYNC, modelNames: string[]): boolean {
+    public getImportedFiles(filePath: string): string[] {
         const code = fs.readFileSync(filePath, 'utf-8');
+        const ast = parser.parse(code, { sourceType: 'module', plugins: ['typescript'] });
+        const importedFiles: string[] = [];
+        const traverseFn = (traverse as any).default || traverse;
```

Comment:
I know it is unrelated to this PR but I've always wondered why does this `.default` thing is doing and why we are not using traverse directly

Reply:
I think during tests traverse wasn't available for some reason